crates/biome_cli/src/execute/traverse.rs

```diff
@@ -713,11 +713,7 @@ impl<'ctx, 'app> TraversalContext for TraversalOptions<'ctx, 'app> {
 
         let file_features = self.workspace.file_features(SupportsFeatureParams {
             path: biome_path.clone(),
-            feature: FeaturesBuilder::new()
-                .with_linter()
-                .with_formatter()
-                .with_organize_imports()
-                .build(),
+            feature: self.execution.to_features(),
```

Comment:
I'd like to change this field to plural form (`features`) as well for clarity when reading the code. :)